app/models/compute_resources/foreman/model/vmware.rb

```diff
@@ -449,6 +449,8 @@ def parse_args(args)
       add_cdrom = args.delete(:add_cdrom)
       args[:cdroms] = [new_cdrom] if add_cdrom == '1'
 
+      args[:volumes]&.each { |v| v[:eager_zero] = v[:eagerzero] }
```

Comment:
Rather than moving this around, can't we change Foreman to use the same one as fog-vsphere does? I didn't look at the code so I don't know the answer to this.

Reply:
This should be possible but this leads to a break with the code convention of using camel case. 
Suggestions?